microprofile/opentracing-extension/src/test/resources/org/wildfly/extension/microprofile/opentracing/subsystem_1_0.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
This actually needs to be removed. The test automatically adds this so it ends up in the file twice which is why the unit test is failing.